### ISSUE TITLE
Site Lacks All Functionality on Older Browsers

### ISSUE THREAD

**Issue by mswider** [chromezoneeths/ethsbell-rewrite#88]
**Description:**
The entirety of ETHSBell is non-functional on older browsers. The only visible content is the static content from the HTML files, and no dynamic content anywhere on the site renders. This makes the site entirely useless as no data whatsoever is displayed.

**To Reproduce:**

1. Use an older browser (Ex: Chrome on iOS 12)
2. Go to [ethsbell.app](https://ethsbell.app/)
3. Notice that page displays nothing except for the header
4. Go to any other page on the site and notice that it is not possible to interact with them or view any information

**Expected behavior**
The site functions as expected with data being shown and pages being interactive.

**Screenshots:**
<img src="https://user-images.githubusercontent.com/37093293/143927802-7b5ae2ca-59d4-423f-afc5-bee913fbcc6d.PNG" width="375px" />
<img src="https://user-images.githubusercontent.com/37093293/143927821-684ef24d-4cbd-45aa-860b-2fffb5c62be5.PNG" width="375px" />
<img src="https://user-images.githubusercontent.com/37093293/143927832-8a8cd7e1-ff89-4f1f-9369-5b1c2955efc7.PNG" width="375px" />
<img src="https://user-images.githubusercontent.com/37093293/143927863-f536a1fc-7202-41c3-86db-8697dbc3290d.PNG" width="375px" />

**Server Version**
Unfortunately, as the version check endpoint is currently broken, its content is meaningless.
It looks like the version which introduced the bug is `v2.0.0`

**OS/Browser Info**
[iPhone 6 running iOS 12.5](https://www.whatsmybrowser.org/b/LG7T3)

**Additional context**
The problem seems to be something to do with the change to the background color, which would have to be using some new features of JS, but that might be wrong.

**Comment by mswider** [comment]
I've looked through the code to try and find the problem and I noticed that [on line 108 of /frontend/index.js](https://github.com/chromezoneeths/ethsbell-rewrite/blob/9c078f11e9c64b7ad4d4913c46f1de93b0a72b18/frontend/index.js#L108) the [logical nullish assignment operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment) (??=) is used. 
This operator is pretty new to JS, being introduced in Chrome 85 and Safari 14 (iOS 14). Using this operator would cause an error in older browsers like mine, which runs on iOS 12. 

I don't think the code using it is used by pages other than the main page, but it would still definitely cause issues for some people.

**Comment by asportnoy** [comment]
Ah, I thought Parcel would transpile that for me. Turns out I need to implement [this](https://parceljs.org/languages/javascript/#browser-compatibility).

**Comment by asportnoy** [comment]
tl;dr add `"browserslist": "defaults"` to `package.json`

I'll take care of this in the morning.

**Comment by mswider** [comment]
Unfortunately the issue persists, somehow. 
I'll take a closer look at it later but I suspect that `frontend/global/helpers.js` is to blame since it's the only thing used by every page, and they're all still broken.

**Comment by mswider** [comment]
(that was posted before but in the wrong issue)

**Comment by mswider** [comment]
Well anyways the **??=** is still in the build
![image](https://user-images.githubusercontent.com/37093293/144331836-690f5cc0-3faf-4c6c-97e6-77cf90b28b93.png)

**Comment by asportnoy** [comment]
Looks like I forgot to copy in the browserslist config into the build which is what was causing that to continue. I'm running docker locally to test it and then I'll open another PR. When I do the building outside of docker there are no issues and it transpiles correctly.

**Comment by asportnoy** [comment]
@mswider I can confirm that the site is now correctly transpiled. Please confirm that the issue is solved.

**Comment by mswider** [comment]
I can confirm that the issue is resolved - all pages now work as intended.
